src/Components/ManageCard/reducer.js

```diff
@@ -0,0 +1,29 @@
+const initialState = {
+     }
+  
+  export default function deleteCardReducer(state = initialState, action) {
+    const { payload, type } = action;
+  
+    switch (type) {
+      case 'DELETECARD_FULFILLED': {
+        return {
+        }
+      }
+
+      case 'DELETECARD_INPUT': {
+        return {
+          ...state,
+          ...payload
+        }
+      }
+     
+      case 'DELETECARD_REJECTED': {
```

Comment:
You can remove this action, it will default to the `default` case

Reply:
Removed.